app/Console/Commands/RebuildImages.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Poniverse\Ponyfm\Console\Commands;
+
+use Illuminate\Console\Command;
+use Poniverse\Ponyfm\Models\Image;
+use Symfony\Component\HttpFoundation\File\File;
+
+class RebuildImages extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rebuild:images';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Resizes all images to fit the specifications in Models/Image';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $images = Image::all();
+        foreach ($images as $image) {
+            $this->info("Regenerating images for id:".$image->id. " (".$image->filename.")");
+            $image->clearExisting();
```

Comment:
Can you overwrite the images one by by one as they're regenerated instead of deleting all the resized versions before regenerating them? Deleting them first means a user may run into HTTP request errors while the rebuild is ongoing, which seems avoidable.